### PR TITLE
Remove unused metrics from Prometheus remote write to Grafana Cloud

### PR DESCRIPTION
**Pull request by alvarof2**
### Description

Remove unused metrics from Prometheus remote write to Grafana Cloud. See this https://github.com/celo-org/infrastructure/issues/305#issuecomment-1152204841. This should account for ~24000 [metrics dropped](https://clabs.grafana.net/explore?orgId=1&left=%7B%22datasource%22:%22grafanacloud-clabs-prom%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22count(%5Cn%20%20%20%20%7B__name__%3D~%5C%22aggregator_.*%7Cauthentication.*%7Ccadvisor_version_info%7Ccloudprovider_gce.*%7Ccontainer_blkio_device_usage_total%7Ccontainer_cpu_cfs_throttled_seconds_total%7Ccontainer_scrape_error%7Ccontainer_ulimits_soft%7Ccortex_.*%7Ccsi_operations_seconds_.*%7Cgo_memstats_.*%7Cgo_threads.*%7Cgo_info.*%7Cgrpc_client_.*%7Cjaeger_.*%7Ckube_deployment_created%7Ckube_deployment_labels%7Ckube_deployment_metadata_generation%7Ckube_deployment_spec_paused%7Ckube_deployment_spec_replicas%7Ckube_deployment_spec_strategy_rollingupdate_max_surge%7Ckube_deployment_spec_strategy_rollingupdate_max_unavailable%7Ckube_deployment_status_condition%7Ckube_deployment_status_observed_generation%7Ckube_deployment_status_replicas_unavailable%7Ckube_deployment_status_replicas_updated%7Ckube_pod_status_ready%7Ckube_pod_status_scheduled%7Ckube_pod_status_scheduled_time%7Cloki_.*%7Cmachine_cpu_sockets%7Cmachine_cpu_physical_cores%7Cmachine_scrape.*%7Cprocess_cpu_system.*%7Cprocess_cpu_user.*%7Cprocess_disk_reads_total%7Cprocess_disk_writes_total%7Cprocess_heap_bytes%7Cprocess_involuntary_context_switches_total%7Cprocess_io_pagefaults_total%7Cprocess_max_fds%7Cprocess_max_resident_memory_bytes%7Cprocess_noio_pagefaults_total%7Cprocess_open_fds%7Cprocess_signals_delivered_total%7Cprocess_start_time_seconds%7Cprocess_swaps_total%7Cprocess_threads_total%7Cprocess_uptime_seconds%7Cprocess_virtual_memory_bytes%7Cprocess_virtual_memory_max_bytes%7Cprocess_voluntary_context_switches_total%7Cpromhttp_.*%7Cnodejs_active_.*%5C%22%7D)%22,%22range%22:true,%22hide%22:false%7D,%7B%22refId%22:%22B%22,%22expr%22:%22%22,%22hide%22:false%7D%5D,%22range%22:%7B%22from%22:%22now-15m%22,%22to%22:%22now%22%7D%7D) if applied to all clusters ([counting all stacks we are ~17000 above the limit](https://clabs.grafana.net/explore?orgId=1&left=%7B%22datasource%22:%22grafanacloud-usage%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22grafanacloud_org_metrics_included_series%20-%20grafanacloud_org_metrics_billable_series%22,%22editorMode%22:%22code%22,%22range%22:false,%22instant%22:true,%22exemplar%22:false%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D).)

### Other changes

N/A

### Tested

`helm template -f values --validate` works

`helm diff upgrade prometheus-stackdriver /home/alvaro/repos/work/Celo/celo-monorepo/packages/helm-charts/prometheus-stackdriver -f values.yaml -n prometheus` in alfajores cluster renders expected output:

```
          - action: drop
-           regex: (aadpodidentity.*|apiserver_.+|authentication.*|certmanager.*|configconnector.*|container_blkio.*|container_cpu_load_average_10s|container_cpu_system_seconds_total|container_cpu_user_seconds_total|container_file_.+|container_fs_.+|container_memory_[^w].*|container_last_.+|container_network_.+|container_processes|container_sockets|container_spec_.+|container_start_.+|container_tasks_state|container_threads|container_threads_max|container_ulimits.*|coredns.*|csi_operations.*|erlang_.+|etcd_.+|go_gc_.*|go_memstats_gc_.*|go_memstats_last_gc_time_seconds|go_memstats_next_gc_bytes|kubedns.*|kube_configmap.*|kube_endpoint.*|kube_job.*|kube_namespace.*|kube_node_status_condition|kube_persistentvolume.*|kube_pod_[^cs].+|kube_pod_status_phase|kube_pod_container_[^r].+|kube_pod_container_status_last_terminated_reason|kube_pod_container_status_terminated_reason|kube_pod_container_status_waiting_reason|kube_replicaset.*|kube_resourcequota.*|kube_secret_.+|kube_service.*|kubelet_[^v].+|net_conntrack_.*|nginx_.+|nodejs_gc_.*|phoenix_.+|prometheus.*|python.*|relay_nodejs_gc_.*|rest_client_.+|scrape.*|state_.+|storage_.+|workqueue_.+)
+           regex: (aadpodidentity.*|aggregator_.*|apiserver_.+|authentication.*|cadvisor_version_info|certmanager.*|cloudprovider_gce.*|configconnector.*|container_blkio.*|container_cpu_cfs_throttled_seconds_total|container_cpu_load_average_10s|container_cpu_system_seconds_total|container_cpu_user_seconds_total|container_file_.+|container_fs_.+|container_memory_[^w].*|container_last_.+|container_network_.+|container_processes|container_scrape_error|container_sockets|container_spec_.+|container_start_.+|container_tasks_state|container_threads|container_threads_max|container_ulimits.*|coredns.*|cortex.*|csi_operations.*|erlang_.+|etcd_.+|go_gc_.*|go_memstats_.*|go_threads.*|go_info.*|grpc_client_.*|jaeger_.*|kubedns.*|kube_configmap.*|kube_deployment_created|kube_deployment_labels|kube_deployment_metadata_generation|kube_deployment_spec_paused|kube_deployment_spec_replicas|kube_deployment_spec_strategy_rollingupdate_max_surge|kube_deployment_spec_strategy_rollingupdate_max_unavailable|kube_deployment_status_condition|kube_deployment_status_observed_generation|kube_deployment_status_replicas_unavailable|kube_deployment_status_replicas_updated|kube_endpoint.*|kube_job.*|kube_namespace.*|kube_node_status_condition|kube_persistentvolume.*|kube_pod_[^cs].+|kube_pod_status_phase|kube_pod_container_[^r].+|kube_pod_container_status_last_terminated_reason|kube_pod_container_status_terminated_reason|kube_pod_container_status_waiting_reason|kube_replicaset.*|kube_resourcequota.*|kube_secret_.+|kube_service.*|kubelet_[^v].+|loki.*|machine_cpu_sockets|machine_cpu_physical_cores|machine_scrape.*|net_conntrack_.*|nginx_.+|nodejs_active_.*|nodejs_gc_.*|phoenix_.+|process_cpu_system.*|process_cpu_user.*|process_disk_reads_total|process_disk_writes_total|process_heap_bytes|process_involuntary_context_switches_total|process_io_pagefaults_total|process_max_fds|process_max_resident_memory_bytes|process_noio_pagefaults_total|process_open_fds|process_signals_delivered_total|process_start_time_seconds|process_swaps_total|process_threads_total|process_uptime_seconds|process_virtual_memory_bytes|process_virtual_memory_max_bytes|process_voluntary_context_switches_total|prometheus.*|promhttp_.*|python.*|relay_nodejs_gc_.*|rest_client_.+|scrape.*|state_.+|storage_.+|workqueue_.+)
            source_labels:
            - __name__
```

### Related issues

- Fixes [infrastructure #305](https://github.com/celo-org/infrastructure/issues/305)

### Backwards compatibility

N/A

### Documentation

N/A

### CODE REVIEW

**Review by lvpeschke — APPROVED** [review]

File: packages/helm-charts/prometheus-stackdriver/values.yaml [hunk 1]
```diff
@@ -181,6 +185,10 @@ remote_write:
               |nginx_.+\
               |nodejs_active_.*\
               |nodejs_gc_.*\
+              |oracle_action_duration_bucket\
+              |oracle_exchange_api_request_duration_seconds_bucket\
+              |oracle_exchange_api_request_error_count\
+              |oracle_ticker_property\
```
Comment (lvpeschke) [inline packages/helm-charts/prometheus-stackdriver/values.yaml:191]: @pedro-clabs or @martinvol - do you need these in grafana cloud?

File: packages/helm-charts/prometheus-stackdriver/values.yaml [hunk 2]
```diff
@@ -136,6 +136,7 @@ remote_write:
               |coredns.*\
               |cortex.*\
               |csi_operations.*\
+              |eksportisto_step_duration_bucket\
```
Comment (lvpeschke) [inline packages/helm-charts/prometheus-stackdriver/values.yaml:139]: @pedro-clabs or @martinvol - do you need this in grafana cloud?

**Comment by alvarof2** [comment]
Added disabling stackdriver [for ODIS](https://github.com/celo-org/infrastructure/issues/307#issuecomment-1156563757)

File: packages/helm-charts/prometheus-stackdriver/values.yaml (hunk 2 above)
Comment (pedro-clabs) [reply packages/helm-charts/prometheus-stackdriver/values.yaml:139]: We are using `eksportisto_step_duration_sum` for sure. If it's possible to filter the bucket part of a histogram metric and leave the sum, I'm not opposed.

File: packages/helm-charts/prometheus-stackdriver/values.yaml (hunk 1 above)
Comment (pedro-clabs) [reply packages/helm-charts/prometheus-stackdriver/values.yaml:191]: `oracle_ticker_property` is very much needed. The others we could do without.

**Review by pedro-clabs — COMMENTED** [review]
Thanks, Álvaro and Victoria!

**Comment by alvarof2** [comment]
@pedro-clabs `eksportisto_step_duration_sum` is still [available](https://clabs.grafana.net/explore?orgId=1&left=%7B%22datasource%22:%22grafanacloud-clabs-prom%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22eksportisto_step_duration_sum%22%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D).

For the `oracle_ticker_property`, it was reported as not being used by Grafana tools. Just to be on the safe side and check the functioning of these tools, are you using it, or just planning to use it?? But I'll update the PR nevertheless, thanx!!

**Comment by pedro-clabs** [comment]
> For the `oracle_ticker_property`, it was reported as not being used by Grafana tools. Just to be on the safe side and check the functioning of these tools, are you using it, or just planning to use it??

We used to use, but we have since moved to `oracle_price_source` -- so please feel free to remove this one. Thanks for checking!

**Comment by alvarof2** [comment]
> > For the `oracle_ticker_property`, it was reported as not being used by Grafana tools. Just to be on the safe side and check the functioning of these tools, are you using it, or just planning to use it??
> 
> We used to use, but we have since moved to `oracle_price_source` -- so please feel free to remove this one. Thanks for checking!

Great, so we can remove both of them :)

**Comment by alvarof2** [comment]
Sorry for spam @lvpeschke @pedro-clabs :pray: but I do not have write permissions on this repo to merge. If any of you could do it when the `master` up-to-date merge passes I'll be grateful so that this does not end up forgotten.

File: packages/helm-charts/prometheus-stackdriver/values.yaml (hunk 1 above)
Comment (lvpeschke) [reply packages/helm-charts/prometheus-stackdriver/values.yaml:191]: Thanks!

**Pull request merged**